### DISCUSSION TITLE
Request failed with status code 502 for command xxx

### DISCUSSION THREAD

**Discussion by ghost** [mrbungle64/ioBroker.ecovacs-deebot#442] [Q&A] [answered]
Hi,
habe ständig Warnungen im Protokoll mit dem Code 502, allerdings gs das Kommando unterscheidet sich.
Noch jemand mit diesem Problem hier?
![Screenshot_20230301_213210_Chrome](https://user-images.githubusercontent.com/60137913/222258654-f263e165-910f-4b29-819d-90a1aa12aaa1.jpg)

**Comment by mrbungle64** [comment, marked as answer]
https://github.com/mrbungle64/ioBroker.ecovacs-deebot/issues/439
